src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    <>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+    </>
+
+    <p className="Person__partner">
+      {person.isMarried && person.sex === 'm' && (
+        <>{`${person.partnerName} is my wife`}</>
+      )}
+      {person.isMarried && person.sex === 'f' && (
+        <>{`${person.partnerName} is my husband`}</>
+      )}
+      {!person.isMarried && <>I am not married</>}
```

Comment:
You've repeated some part of code
Will be better to define the info about the partner initially, should it be `wife` or `husband`
And then use this data while preparing the content